build.sbt

```
@@ -71,13 +71,17 @@ lazy val rules = project
   .in(file("scalafix-rules"))
   .settings(
     moduleName := "scalafix-rules",
+    buildInfoKeys ++= Seq[BuildInfoKey](
+      "supportedScalaVersionforRules" -> supportedScalaVersionforRules
```

since this is local to the rules module, I don't think the suffix is required for the key
```suggestion
      "supportedScalaVersions" -> supportedScalaVersionforRules
```